client/js/screens/Account/Account.js

```diff
@@ -1,36 +1,66 @@
 import React from "react";
-import { View, Text, Button, AsyncStorage } from "react-native";
+import {
+  View,
+  Text,
+  ScrollView,
+  Image,
+  AsyncStorage,
+  TouchableOpacity
+} from "react-native";
+import styles from "./styles";
+import {withNavigation} from "react-navigation";
 
 const Accounts = props => {
   _signOutAsync = async () => {
     await AsyncStorage.clear();
     props.navigation.navigate("Auth");
   };
-
   return (
-    <View>
-      <Button
-        style={{ borderStyle: "solid", borderWidth: "1" }}
-        title="Change Password"
-        onPress={() => {}}
-      />
-      <Button
-        style={{ borderStyle: "solid", borderWidth: "1" }}
-        title="Disconnet from Cool It Program"
-        onPress={() => {}}
-      />
-      <Button
-        style={{ borderStyle: "solid", borderWidth: "1" }}
-        title="Deactivate Account"
-        onPress={() => {}}
-      />
-      <Button
-        style={{ borderStyle: "solid", borderWidth: "1" }}
-        title="Sign Out"
-        onPress={_signOutAsync}
-      />
-    </View>
+    <ScrollView contentContainerStyle={styles.contentContainer}>
+      <View style={styles.container}>
+        <View style={styles.mainContent}>
+          <View style={styles.itemSeparator} />
+          <Text style={styles.description}>Account</Text>
+          <View style={styles.itemBottomSeparator} />
+        </View>
+
+        <View style={styles.menu}>
+          <View style={styles.buttonSeparator} />
+          <TouchableOpacity onPress={() => {}} style={styles.logIn}>
+            <Text style={styles.buttonText}>Change Password</Text>
+          </TouchableOpacity>
+          <View style={styles.buttonSeparator} />
+
+          <View style={styles.buttonSeparator} />
+          <TouchableOpacity onPress={() => {}} style={styles.logIn}>
+            <Text style={styles.buttonText}>
+              Disconnect My Account From The Cool It Program
+            </Text>
+          </TouchableOpacity>
+          <View style={styles.buttonSeparator} />
+
+          <View style={styles.buttonSeparator} />
+          <TouchableOpacity onPress={() => {}} style={styles.logIn}>
+            <Text style={styles.buttonText}>Deactivate Account</Text>
+          </TouchableOpacity>
+          <View style={styles.buttonSeparator} />
+
+          <View style={styles.buttonSeparator} />
+          <TouchableOpacity onPress={_signOutAsync} style={styles.logIn}>
+            <Text style={styles.buttonText}>Log out </Text>
+          </TouchableOpacity>
+          <View style={styles.buttonSeparator} />
+        </View>
+
+        <View style={styles.bottom}>
+          <Image
+            source={require("../../assets/images/valley.png")}
+            style={styles.valley}
+          />
+        </View>
+      </View>
+    </ScrollView>
   );
 };
 
-export default Accounts;
+export default withNavigation(Accounts);
```

Comment:
you do not need withNavigation here. Pass navigation prop from the container. Thank you.